package.json

```diff
@@ -47,13 +51,14 @@
     "codecov": "^3.5.0",
     "gts": "^2.0.0",
     "js-green-licenses": "^1.0.0",
+    "microbundle": "^0.11.0",
     "mocha": "^7.0.0",
     "nock": "^12.0.0",
     "semantic-release": "^17.0.4",
     "typescript": "~3.8.0"
   },
   "files": [
-    "build/src"
```

Comment:
... we don't need both?

Reply:
No, as far as I'm aware of `dist/` is enough.

The `prepare` script bundles only into `dist/`.
And `compile` is only execute at `pretest` and compiles into `build/`.